src/showcase/app/business/business-examples/accordion-examples/basic-accordion-example/basic-accordion-example.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
```

Comment:
To reduce complexity i'd suggest to only use reactiveForms for disabled, panelMode and hideToggle. Can also be done later...